pkg/utils/memory_resources.go

```diff
@@ -67,6 +67,10 @@ func GetNumaMemoryResources() (NumaMemoryResources, error) {
 		info[corev1.ResourceMemory] = nodeTotalMemory
 
 		// Get hugepages
+		_, err = os.Stat(filepath.Join(sysBusNodeBasepath, numaNode, "hugepages"))
```

Comment:
nit: I suggest to put the path in a variable and use that same value in the `getHugepagesBytes()` below to make it evident and sure that we actually use the same path

/cc @PiotrProkop @ffromani

Reply:
ping @freelizhun

Reply:
is this ok ? @marquiz
```golang
               // Get hugepages
		huagePagePath := filepath.Join(sysBusNodeBasepath, numaNode, "hugepages")
		_, err = os.Stat(huagePagePath)
		if err != nil && os.IsNotExist(err) {
			continue
		}
		hugepageBytes, err := getHugepagesBytes(huagePagePath)
```

Reply:
Yeah, sure